src/plugins/label/styles.scss

```diff
@@ -0,0 +1,29 @@
+.ExpandableTestWidget {
+    display: flex;
+    align-items: center;
+    justify-content: center;
```

Comment:
In Anori titles are always left-aligned, could you update styles to not center the title?